### PR TITLE
WT-5510 Enable more tests in durable history branch

### PR DESCRIPTION
**Pull request by tetsuo-cpp**
I was going to enable `test_hs01` as per @luke-pearson's ticket but I realized that quite a few tests are now passing.

### CODE REVIEW

**Comment by agorrod** [comment]
This seems broader than the description. Is there another ticket tracking test_gc02 failure?

**Comment by tetsuo-cpp** [comment]
> This seems broader than the description. Is there another ticket tracking test_gc02 failure?

I realized there are other tests that are passing now so I'll enable them too. I will adjust the title when I've got the build stable.

I will make a ticket for `gc02` shortly.

File: test/suite/test_gc02.py [hunk 1]
```diff
@@ -39,6 +40,7 @@ class test_gc02(test_gc_base):
     conn_config = 'cache_size=1GB,log=(enabled),statistics=(all)'
     session_config = 'isolation=snapshot'
 
+    @unittest.skip("Temporarily disabled")
```
Comment (quchenhao) [inline test/suite/test_gc02.py]: no need to disable this

**Comment by tetsuo-cpp** [comment]
@agorrod It looks like `gc02` will be fixed shortly by @quchenhao's incoming change so I've decided not to create a separate ticket.

File: test/suite/test_gc02.py (hunk 1 above)
Comment (tetsuo-cpp) [reply test/suite/test_gc02.py]: It's not passing in my branch so I'll leave that to you.

**Pull request merged**